human_sim/Goal goal
---
move_base_msgs/MoveBaseGoal[] actions
